ParkingZoneApp/Services/ParkingSlotService.cs

```diff
@@ -15,12 +16,29 @@ public IEnumerable<ParkingSlot> GetByParkingZoneId(int parkingZoneId)
             return ParkingSlots;
         }
 
-        public bool ParkingSlotExits(int ParkingZoneId, int ParkingSlotNumber)
+        public bool ParkingSlotExists(int parkingZoneId, int parkingSlotNumber)
         {
             var ParkingSlots = _repository.GetAll()
-                .Where(x => x.ParkingZoneId == ParkingZoneId &&
-                x.Number == ParkingSlotNumber);
+                .Where(x => x.ParkingZoneId == parkingZoneId &&
+                x.Number == parkingSlotNumber);
             return ParkingSlots.Count() == 0 ? false : true;
         }
+
+        public IEnumerable<ParkingSlot> GetFreeByParkingZoneIdAndPeriod(int parkingZoneId, DateTime startTime, int duration)
+        {
+            var slots = _repository.GetAll()
+                .Where(x => x.ParkingZoneId == parkingZoneId && x.IsAvailableForBooking);
```

Comment:
this logic does not return free slots but only all slots which depend on specific parking zone

Reply:
![image](https://github.com/SamanKingDotNetDeveloper/Parking_Zone_Client/assets/100861214/9c3a5092-f0da-45ab-b964-f3816bfb1db4)
I can not see it in my visual studio is is showing normal